pydicer/preprocess/data.py

```diff
@@ -155,13 +156,17 @@ def preprocess(self):
             except Exception as e:  # pylint: disable=broad-except
                 # Broad except ok here, since we will put these file into a
                 # quarantine location for further inspection.
+                logger.exception(e)
                 logger.error(
```

Comment:
Probably remove the the `.error()` logging of the exception since it's being logged with `.exception()`

Reply:
I think we might need to keep it, so that its easier to see exactly what file caused the error. If we only print the exception we see the stacktrace but not what file caused it...